iocage_lib/ioc_list.py

```diff
@@ -126,11 +126,13 @@ def list_all_quick(self, jails):
 
             uuid = conf["host_hostuuid"]
             ip4 = conf.get('ip4_addr', 'none')
-            dhcp = True if conf.get('dhcp') == 'on' or 'DHCP' in \
-                ip4.upper() else False
+            dhcp = True if iocage_lib.ioc_common.check_truthy(
```

Comment:
This is tricky. I am not sure what is the best way forward with this. This also follows the same pattern as above for ip values.